src/langcheck/augment/_common/_jailbreak_template.py

```diff
@@ -44,6 +47,18 @@ def jailbreak_template_common(
     if templates is None:
         templates = available_templates
 
+    if custom_templates is not None:
+        for template_name, path_to_template in custom_templates:
+            # validation
+            if template_name in available_templates:
+                raise ValueError(f"A template with the name {template_name} already exists!")
+            template_file = Path(path_to_template)
+            if not template_file.exists():
+                raise ValueError(f"Custom template file {path_to_template} does not exist!")
+            if not path_to_template.endswith(".j2"):
+                raise ValueError("The custom template file must be a Jinja2 template file with the extension '.j2'!")
+            available_templates.append(template_name)
```

Comment:
It updates the object passed as the argument directly. Usually `AVAILABLE_JAILBREAK_TEMPLATES` constants in the en/ja modules are used, so as a result your custom `template_name` is added to the list and you cannot call the function with the same `template_name` from the second time because of the validation.

Could you update the code so that this operation doesn't update the referenced `available_templates` object? (probably `available_templates = [*available_templates, template_name]`?)